bytecode/chunk.ts

```diff
@@ -13,7 +14,7 @@ export class Chunk {
   // little challenging, though, and will require more research.
   //
   // For now, I'm using an Array to prototype.
-  public code: OpCode[] = [];
+  public code: Array<OpCode | Byte> = [];
```

Comment:
This type should probably be `Byte[]`, in order to enforce that `OpCode` is a byte enum.